.ci/Jenkinsfile

```
@@ -49,9 +49,12 @@ pipeline {
         cleanup()
         withMageEnv(pkgs: [
           "golang.org/x/lint/golint",
-          "github.com/golang/protobuf/proto",
-          "github.com/golang/protobuf/protoc-gen-go"
+          "google.golang.org/protobuf",
           ]){
+          sh '''
+          curl -sSfL -o protoc.zip https://github.com/protocolbuffers/protobuf/releases/download/v3.12.3/protoc-3.12.3-linux-x86_64.zip
+          unzup protoc.zip
```

Should this be `unzip`? 
```suggestion
          unzip protoc.zip
```

---

I resolved it yesterday, weird you can see it